### PR TITLE
Feature/aos 2760 gorg

### PR DESCRIPTION
**Pull request by martinrodland**

### CODE REVIEW

File: src/main/kotlin/no/skatteetaten/aurora/gorg/extensions/BuildConfig.kt [hunk 1]
```diff
@@ -0,0 +1,17 @@
+package no.skatteetaten.aurora.gorg.extensions
+
+import io.fabric8.openshift.api.model.BuildConfig
+import no.skatteetaten.aurora.gorg.service.BuildConfigResource
+import java.time.Duration
+import java.time.Instant
+
+fun BuildConfig.toResource(now: Instant): BuildConfigResource {
```
Comment (jarlehansen) [inline src/main/kotlin/no/skatteetaten/aurora/gorg/extensions/BuildConfig.kt:8]: Perhaps we can merge some of the files in the `extensions` package?
Seems like many of them work on similar object, from kubernetes-model

File: src/main/kotlin/no/skatteetaten/aurora/gorg/service/OpenShiftService.kt [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+import no.skatteetaten.aurora.gorg.extensions.toResource
+import org.springframework.stereotype.Service
+import java.time.Instant
+
+@Service
+class OpenShiftService(
+    val client: OpenShiftClient
+) {
+
+    fun findTemporaryProjects(now: Instant): List<ProjectResource> =
```
Comment (jarlehansen) [inline src/main/kotlin/no/skatteetaten/aurora/gorg/service/OpenShiftService.kt]: Add Instant.now() as the default argument?

File: src/main/kotlin/no/skatteetaten/aurora/gorg/extensions/BuildConfig.kt (hunk 1 above)
Comment (bjartek) [reply src/main/kotlin/no/skatteetaten/aurora/gorg/extensions/BuildConfig.kt:8]: In other of our projects we put extention methods into a file that is named after the class that we extend. I feel it is easier to find them that way. I am open to joining them on functionality/group though.

File: src/main/kotlin/no/skatteetaten/aurora/gorg/service/OpenShiftService.kt (hunk 2 above)
Comment (bjartek) [reply src/main/kotlin/no/skatteetaten/aurora/gorg/service/OpenShiftService.kt]: That could work, and would make the happy path of the code look cleaner

**Review by jarlehansen — APPROVED** [review]

**Pull request merged**